samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -85,15 +85,9 @@ public SemanticKernelController(IServiceProvider serviceProvider, IConfiguration
         SKContext result = await kernel.RunAsync(contextVariables, function!);
         if (result.ErrorOccurred)
         {
-            // TODO latest NuGets don't have the Detail property on AIException
-            //if (result.LastException is AIException aiException && aiException.Detail is not null)
-            //{
-            //    return this.BadRequest(string.Concat(aiException.Message, " - Detail: " + aiException.Detail));
-            //}
-
-            if (result.LastException is AIException aiException)
+            if (result.LastException is AIException aiException && aiException.Detail is not null)
             {
-                return this.BadRequest(aiException.Message);
+                return this.BadRequest(string.Concat(aiException.Message, " - Detail: " + aiException.Detail));
```

Comment:
I think exception message is enough, exception details have redundant message and adds noise. We can say "check api logs for more details".

Reply:
Totally agree, but in this case we don't get the source of the problem without displaying the details. We only get Something like "BadRequest => blah blah 404". With the details it shows that we have misconfigured the connection to Completion.